application/org.openjdk.jmc.flightrecorder.ui/src/main/resources/org/openjdk/jmc/flightrecorder/ui/messages/internal/messages.properties

```diff
@@ -295,6 +295,8 @@ GarbageCollectionsPage_ROW_METASPACE_DESC=This lane can show various metaspace r
 GarbageCollectionsPage_ROW_PAUSE_PHASES=Pause Phases
 GarbageCollectionsPage_ROW_PAUSE_PHASES_DESC=Phases of the garbage collection pauses
 GarbageCollectionsPage_TIMELINE_SELECTION=Garbage Collections Timeline Selection
+GarbageCollectionsPage_USED_HEAP_AFTERGC=Used Heap After GC
```

Comment:
Underscore between AFTER and GC, and BEFORE and GC. See e.g. HEAP_SUMMARY_BEFORE_GC. Otherwise all looks fine! 👍